backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -966,7 +966,10 @@ class TimezoneSchema(marshmallow.Schema):
 
 class AboutSchema(marshmallow.Schema):
     name = StrippedString(example="Tracim", description="Software name")
-    version = StrippedString(example="2.0", allow_none=True, description="Version of Tracim")
+    version = StrippedString(example="2.6", allow_none=True, description="Version of Tracim")
+    build_version = StrippedString(
+        example="release_02.06.00", allow_none=True, description="Build Version of Tracim"
+    )
```

Comment:
Why `version` and `build_version` can be `None` ?

Reply:
fixed.